### DISCUSSION TITLE
How to unify color bar of different IPDFs

### DISCUSSION THREAD

**Discussion by hwhouse** [mtex-toolbox/mtex#753] [Ask Anything]
Hi
I was trying to plot IPDFs relative to X, Y, Z direction in the specimen system with the next line:
plotIPDF(ori, [vector3d.X,vector3d.Y,vector3d.Z],'contourf'); 
mtexColorbar;
and the result is like this:
![untitled](https://user-images.githubusercontent.com/34276511/107116013-87161600-68ab-11eb-88c6-2e30f926e3c7.png)
I wonder is there any method to unify the color bar of all three figures

best regards
Wang Hao

**Comment by lfgmorales** [comment]
Hi

Have a look here

https://mtex-toolbox.github.io/Annotations.html

Cheers

Luiz


Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 6 Feb 2021, at 11:48, hwhouse <notifications@github.com> wrote:

﻿

Hi
I was trying to plot IPDFs relative to X, Y, Z direction in the specimen system with the next line:
plotIPDF(ori, [vector3d.X,vector3d.Y,vector3d.Z],'contourf');
mtexColorbar;
and the result is like this:
[untitled]<https://user-images.githubusercontent.com/34276511/107116013-87161600-68ab-11eb-88c6-2e30f926e3c7.png>
I wonder is there any method to unify the color bar of all three figures

best regards
Wang Hao

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/753>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN6XC5IRJ6IIRB25RWTS5UNBHANCNFSM4XGE7K5A>.

[ { "@context": "http://schema.org", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/753", "url": "https://github.com/mtex-toolbox/mtex/discussions/753", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
